packages/core/contracts/oracle/implementation/Voting.sol

```diff
@@ -254,6 +264,7 @@ contract Voting is
         uint256 blockTime = getCurrentTime();
         require(time <= blockTime, "Can only request in past");
         require(_getIdentifierWhitelist().isIdentifierSupported(identifier), "Unsupported identifier request");
+        require(ancillaryData.length <= ancillaryBytesLimit, "Invalid ancillary data");
```

Comment:
Should we add the same require to the OO?

Reply:
yeah its a bit redundant bytecode-wise but it makes it more readable. Do you think a comment in the OO is not enough?

Reply:
Ah yes good catch, done!